internal/config/audit.go

```diff
@@ -22,7 +22,7 @@ type AuditConfig struct {
 
 // Enabled returns true if any nested sink is enabled
 func (c AuditConfig) Enabled() bool {
-	return c.Sinks.Log.Enabled || c.Sinks.Webhook.Enabled || c.Sinks.Cloud.Enabled || c.Sinks.Kafka.Enabled
```

Comment:
cloud audit sink hasnt been supported for a while, figured might as well remove it now